packages/lit-dev-content/site/tutorials/content/intro-to-lit/05.md

```diff
@@ -38,21 +38,32 @@ You can use standard JavaScript in Lit expressions to create conditional or repe
     }
 
     addToDo() {
-      this.listItems.push({text: this.input.value, completed: false});
+      this.listItems = [...this.listItems,
+          {text: this.input.value, completed: false}];
```

Comment:
I still personally disagree with this method being the one we choose to highlight.

Especially since it's an input `@property` to the component – any references the user of this component hands to this component is no longer the one used inside this component. The user then has to query the component again and get the new refernce. Then there's idea of cloning objects etc.

Reply:
Yeah, I can see the argument here with regard to it being an input property. It would probably make more sense for it to *not* be an input property, since the component manipulates it. 

@kevinpschaaf and @sorvell both felt we should not be recommending `requestUpdate()` for most use cases.

Reply:
This is more of a "alright but I'mma complain the entire time" feel free to ignore the noise 🌬️

Reply:
I think we should make this `@state` and then split this into 2 steps. One just intro'ing the `@state` and adding the list and the other doing the `add` stuff.

Reply:
If it's too big of a change to split out another step, an aside that says something like this seems fine:

> Notice that `listItems` uses the `@state()` decorator. That makes it reactive just like `@property()`, but is internal to our element, not part of the public API of our element. Read more about [Internal reactive state](https://lit.dev/docs/components/properties/#internal-reactive-state).

Reply:
I think this latter should work. It'll just be a matter of making sure I fix all the references to listItems. Which should probably also become private or protected `_listItems`?

Reply:
Made this change, and added a new paragraph about state. The asides just seem way to distracting to put inline.